app/models/manageiq/providers/azure/inventory/collector/container_manager.rb

```diff
@@ -1,3 +1,3 @@
-class ManageIQ::Providers::Azure::Inventory::Collector::ContainerManager < ManageIQ::Providers::Kubernetes::Inventory::Collector::               ContainerManager
+class ManageIQ::Providers::Azure::Inventory::Collector::ContainerManager < ManageIQ::Providers::Kubernetes::Inventory::Collector::ContainerManager
```

Comment:
:scream: Really funky vim copy&paste issue here, can't believe I missed this
Amazingly it actually worked though